tox.ini

```diff
@@ -14,7 +14,9 @@ python =
 
 [testenv]
 deps =
-    pytest
+    pytest5: pytest>=5.0,<5.9
+    pytest6: pytest>=6.0,<6.9
```

Comment:
why limit to 5.9 or 6.9?

Reply:
Right, it should be 6.0 and 7.0.

Reply:
Fixed: https://github.com/flexmock/flexmock/pull/51/commits/2092441f9ed60151666e03db0c9008a6cde2bfaf